src/Test/_project.json

```diff
@@ -29,6 +29,7 @@
     "dependencies": {
         "NETStandard.Library": "1.0.0-rc3-23829",
         "Microsoft.NETCore.TestHost": "1.0.0-rc3-23829",
+        "System.Buffers" : "4.0.0-rc3-23829",
```

Comment:
I got an error of missing runtime DLL System.Buffers while running the tests. This fixed that issue, but I have not researched it further.